app/views/users/destroy.html.erb

```diff
@@ -0,0 +1,22 @@
+<%- model_class = User -%>
+
+<div class="page-header">
```

Comment:
since you have this page now, the confirmation dialog when clicking "delete this account" feels like an unneeded additional layer

Reply:
Done